pages/doc/integrations_tas_custom_app_metrics_howto.md

```diff
@@ -0,0 +1,75 @@
+---
+title: How to Monitor Custom App Metrics on TAS with Tanzu Observability
+keywords:
+tags: [integrations]
+sidebar: doc_sidebar
+permalink: integrations_tas_custom_app_metrics_howto.html
+summary: How to get your custom application metrics to appear in Tanzu Observability by Wavefront.
+---
+
+Out of the box, the Wavefront Nozzle for TAS provides metrics for monitoring the TAS platform. For monitoring applications running **on** the platform, the Nozzle supports two approaches:
+
+1. **Metrics Registrar:** a **pull-based** model based on the **Prometheus metrics format**
+1. **Wavefront Service Broker:** a **push-based** model where applications send metrics directly to a Wavefront Proxy.
+
+**We recommend the Metrics Registrar approach** as a simpler and more predictable pattern, but teams may choose either option. Both have drawbacks, and for some workloads only the push method can be used in practice.
```

Comment:
The **Metric Registrar** model is recommended, because it is a simpler and more predictable pattern, but you can choose either option. Both models have drawbacks, and for some workloads only the push-based method can be used in practice.